Source_Code/Server/ClueLessServer.java

```diff
@@ -351,6 +357,9 @@ public void processGameStartRequest(GameStartRequest gsr)
                     ServState = ServerState.ActiveGame;
                     sendToAllPlayers(new GameStart());
                     
+                    // Update player objects with all the cards in the deck
+                    CardDeck.shuffleAndAssignCards( PlayerList );
+                    
```

Comment:
Need to actually send the assigned cards to the clients. This is probably the place to do it? Don't feel super strongly if you think it should go elsewhere

Reply:
The sending will go here. 
The actual code to do that will be pushed shortly.